content/docs/command-reference/move.md

```diff
@@ -4,6 +4,10 @@ Rename a file or a directory and modify the corresponding `.dvc` file (see
 `dvc add`) to reflect the change. If the file or directory has the same name as
 the `.dvc` file, it also renames it.
 
+Renaming is supported only for data source files and directories (creaded by
+`dvc add` and `dvc import`), not for outputs of pipelines (`dvc run`). The
+pipeline outputs can be renamed only manualy.
+
```

Comment:
Great info. We should move it down to the description though, the intro is supposed to be as short and on-point as possible.

Reply:
Actually this whole ref is still outdated (pre 1.x) unfortunately. dvc.yaml or dvc run are not even mentioned yet so this text is a bit out of context... I.e. you can just leave the PR there and we'll reuse it to update the whole ref. it may just take a few days to get to this.

Reply:
Cc @imhardikj probably a good one to start for you 1.x updates. It should be simple enough and let us practice this side of your project.